test/functorch/test_vmap.py

```diff
@@ -3236,8 +3236,6 @@ def test():
         xfail('broadcast_shapes', ''),  # test runner can't handle non-Tensor ops
         xfail('sparse.sampled_addmm'),  # sparse
         xfail('cross'),  # The default value of dim in op is *very* weird. No wonder it doesn't work
-        xfail('svd', device_type='cuda'),  # not unique, see test_linalg_svd for manual test
-        xfail('linalg.svd', device_type='cuda'),  # not unique, see test_linalg_svd for manual test
```

Comment:
Does linalg.svd produce the same result for a matrix and a batch of matrices containing that matrix?

Previously the answer was no, hence the xfail. If with the new implementation this is guaranteed then removing the xfail is fine; if it's not guaranteed then it might be better to change this to a skip

Reply:
Let's make it a skip, someone might add a larger input size to OpInfo in the future and be confused when the test fails

Reply:
Actually, nevermind, the answer to this question is now always yes, as it'll always use the same algorithm regardless of whether the matrix is a batch or not.

Reply:
I'm removing the xfail and I won't tag it as a skip.

Reply:
SGTM